frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx

```diff
@@ -255,7 +255,7 @@ export function NotebookSelectButton({ children, onNotebookOpened, ...props }: N
                 }
             }}
         >
-            {children ?? 'Add to notebook'}
+            {children ?? 'Notebooks'}
```

Comment:
This affects quite a few places. @PostHog/team-replay hope you're ok with this language change?

Reply:
I'd be tempted to put the original text in a tooltip for the button
But we can also change it, see if we get user feedback, and do a graph of whether people are clicking it.

If folk are still clicking it then we're all good :)